tests/test_data.yml

```diff
@@ -57,3 +57,17 @@ pinterest:
     - "steve"
   available_usernames:
     - "zNRe3jx3isA8CoM"
+openhub:
+  taken_usernames:
+    - "seeeturtle"
+    - "andrewda"
+    - "nibnalin"
+  available_usernames:
+    - "zNRe3jx3isA8CoM"
+deviantart:
```

Comment:
insert in roughly alpha order; i.e. `deviantart` at the beginning. `openhub` before `pinterest`